static/js/apps/nl_interface/query_examples.tsx

```diff
@@ -18,248 +18,47 @@
-    "Show me cities with the highest increase in home prices in the last 3 years?",
-    "What percentage of homes in the US are owned by foreign residents?",
-    "How much of Brooklyn’s apartments are rent controlled?",
-    "What percent of millennials own a home?",
-  ],
-  sustainability: [
-    "Show me wet bulb temperatures for India",
-    "Which areas of the US have a composting system in place?",
-    "What is the predicted sea level rise over the next 15 years?",
-    "What factors affect climate change? <general>",
-    "Show me global temperature increase predictions over the next decade",
-    "How many states encourage recycling?",
-    "I want to see global warming trends",
-    "How many more people have electric cars instead of gas cars compared to 20 years ago?",
-    "Which cities have switched to using compostable plastics?",
-    "Which countries have the smallest carbon footprint?",
-    "How has the rate of the ice caps melting changed over the century?",
-    "How has the EU government contributed to improving sustainability over the last decade? <general>",
-    "How have poverty rates in South America changed since 2010?",
-    "Countries with the best universal health care",
-    "What is the impact of pollution on ocean life in Tulum?",
-    "How much of the African population has access to clean drinking water?",
-    "Which countries have the best access to affordable and clean energy?",
-    "Extinction rate of endangered animals over time",
-    "How has the climate in my area shifted over the past 50 years?",
-    "How much has malnutrition decreased over time in Latin America?",
-    "Which US counties have banned plastic straws? <general>",
-    "Which countries have recently put policies in place to conserve the oceans? <general>",
-    "Show me greenhouse gas emissions levels across the globe",
-  ],
-  biomedical: [
-    "What is the prevalence of heart disease in Texas?",
-    "How does the prevalence of heart disease correlate with projected temperatures in the US?",
-    "What is the prevalence of asthma in Asia?",
-  ],
-};
-
 interface QueryExamplesPropType {
   // Callback function for exmple item clicks.
   onItemClick: (queries: string[]) => void;
 }
 
 export function QueryExamples(props: QueryExamplesPropType): JSX.Element {
+  const [csvData, setCsvData] = useState<Array<Array<string>>>();
+
+  function fetchData(): void {
+    axios.get("/data/nl/topics.csv").then((resp) => {
+      Papa.parse(resp.data, {
+        complete: (result) => {
+          setCsvData(result["data"]);
+        },
+        worker: true,
+      });
+    });
+  }
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
   const topic = getUrlToken("topic");
-  if (!topic || !TOPIC_QUERIES[topic]) {
+  if (!topic) {
     return null;
   }
-  const examples = TOPIC_QUERIES[topic];
+  const examples = [];
+  if (csvData) {
+    for (const row of csvData.slice(1)) {
+      if (row[0] === topic && row[2] === "yes") {
```

Comment:
nit: Be case tolerant and maybe allow "y"?

lower(strip(row[1])) && (lower(strip(row[2])) === "yes" || lower(strip(row[2])) === "y")

Reply:
Done